k8s/upgrades/1.0.0-1.1.0/patch_remove_bdc_finalizer.json

```diff
@@ -0,0 +1,3 @@
+[
+  {"op": "remove", "path": "/metadata/finalizers"}
```

Comment:
'm bit skeptical with deleting finalizers.. If finalizers are removed first and upgrade script breaks, then, we have a BDC without finalizers